gcalcli/argparsers.py

```diff
@@ -250,6 +253,14 @@ def get_argument_parser():
     quick.add_argument("text")
 
     add = sub.add_parser("add", parents=[details_parser, remind_parser])
+    add.add_argument(
+            "--color",
```

Comment:
```suggestion
            "--color", default=None, type=str
```

Reply:
This is a small thing, your style is PEP8 compliant, but I want to maintain consistent style in this file (and project) as much as possible.